fortran/src/H5Pff.F90

```diff
@@ -3470,17 +3470,17 @@ SUBROUTINE h5pset_link_creation_order_f(gcpl_id, crt_order_flags, hdferr)
     INTEGER, INTENT(IN) :: crt_order_flags
     INTEGER, INTENT(OUT) :: hdferr
     INTERFACE
-       INTEGER FUNCTION h5pset_link_creation_order_c(gcpl_id, crt_order_flags) &
-            BIND(C,NAME='h5pset_link_creation_order_c')
-         IMPORT :: HID_T
+       INTEGER FUNCTION h5pset_link_creation_order(gcpl_id, crt_order_flags) &
```

Comment:
Since this is renaming a Fortran API routine, it should probably be mentioned in the release document.

Reply:
The Fortran API name was not changed. The interface changes are transparent to the user and refer to the C API signature.